core/tests/performance_framework/perf_services.py

```diff
@@ -240,8 +240,11 @@ def _setup_driver(self, proxy=None, use_proxy=False):
             chrome_options = webdriver.ChromeOptions()
             # Disable several subsystems which run network requests in the
             # background. This helps reduce noise when measuring network
-            # performance.
+            # performance. Also, disable prerendering by chrome which renders
+            # a page in the background leading to wrong test results.
```

Comment:
Right, yep, there's a principle to be decided upon here:
(a) should these tests match what actually happens when a user loads the page?
(b) or are these tests meant to map specifically to what's happening visibly in a particular page, and ignore additional calls to other things?

I actually lean slightly towards (a), but I'll leave this to you and @BenHenning (and @maitbayev?) to decide since you've discussed the performance tests in more detail and have a better sense of what they should represent conceptually.